sql/item_strfunc.h

```diff
@@ -1251,6 +1251,26 @@ class Item_func_current_user :public Item_func_user
   }
 };
 
+class Item_func_session_user :public Item_func_user
+{
+public:
+  Item_func_session_user(THD *thd):
+    Item_func_user(thd) {}
+  bool fix_fields(THD *thd, Item **ref) override;
+  LEX_CSTRING func_name_cstring() const override
+  {
+    static LEX_CSTRING name= {STRING_WITH_LEN("session_user") };
+    return name;
+  }
+  const char *fully_qualified_func_name() const override
+  { return "session_user()"; }
+  bool check_vcol_func_processor(void *arg) override
```

Comment:
doesn't seem to be necessary? `Item_func_sysconst` (two classes up the hierarchy) has exactly that.